common/djangoapps/static_replace/management/commands/clear_collectstatic_cache.py

```diff
@@ -1,14 +1,21 @@
-###
-### Script for importing courseware from XML format
-###
+"""
+Script for importing courseware from XML format
```

Comment:
It looks like the docstring and help text have nothing to do with the command implementation; copy-pasta'd from another command, I guess?

Reply:
Seems likely. I'll fix it up.